pkg/node/node.go

```diff
@@ -255,6 +257,7 @@ func (n *Node) CalculateSignature(
 				keepAddress.String(),
 				err,
 			)
+			time.Sleep(time.Second)
```

Comment:
Maybe we could extract the value to a constant defined on top of the file and reuse it?

Reply:
Hm, I guess I'm open, but what do you think we should name this constant? We use `time.Second` in 3 different places in slightly different contexts and it is already a constant. Naming `const second = time.Second` feels redundant. Smth general like `contextWaitInterval`?
Also we have similar case in `publishSignature` -> `time.Sleep(1 * time.Minute)`, but this one is used only in 1 place.

Reply:
All the three places are related, maybe we could name it just `retryDelay`?

The value in `publishSignature` is a different story and can remain as is.

Reply:
done